README.md

```diff
@@ -29,9 +28,9 @@ wallet functionality and this was a very intentional design decision.  See the
 blog entry [here](https://blog.conformal.com/btcd-not-your-moms-bitcoin-daemon)
 for more details.  This means you can't actually make or receive payments
 directly with btcd.  That functionality is provided by the
-[btcwallet](https://github.com/btcsuite/btcwallet) and
-[btcgui](https://github.com/btcsuite/btcgui) projects which are both under
-active development.
+[btcwallet - CLI-only](https://github.com/btcsuite/btcwallet) and
+[Paymetheus - Windows-only](https://github.com/btcsuite/Paymetheus) projects
```

Comment:
If the CLI-only line is removed from btcwallet, I think it would read better to put (Windows only) in parentheses after the link.